scripts/get_coco.sh

```diff
@@ -3,14 +3,14 @@
 # Download command: bash ./scripts/get_coco.sh
 
 # Download/unzip labels
-d='./' # unzip directory
+d='./yolov7' # unzip directory
 url=https://github.com/ultralytics/yolov5/releases/download/v1.0/
 f='coco2017labels-segments.zip' # or 'coco2017labels.zip', 68 MB
 echo 'Downloading' $url$f ' ...'
 curl -L $url$f -o $f && unzip -q $f -d $d && rm $f & # download, unzip, remove in background
 
 # Download/unzip images
-d='./coco/images' # unzip directory
+d='./yolov7/coco/images' # unzip directory
```

Comment:
```suggestion
d='./coco/images' # unzip directory
```
(same as above - this is meant to go in the root of the repo, not inside the package)